### DISCUSSION TITLE
Setting the initialColorMode doesn't seem to work

### DISCUSSION THREAD

**Discussion by Indy-Dev** [chakra-ui/chakra-ui#2212] [Q&A] [answered]
Hi,

I'm trying to set the default color mode to dark with this:

```
const overrides = {
  config: {
    initialColorMode: "dark",
    useSystemColorMode: false,
  },
};

export default extendTheme(overrides);
```

But it's not setting the default color mode to dark. Instead, we have this:

`<body class="chakra-ui-light">`

I've triple checked the theme object produced by `extendTheme()` and it look fine.

Any ideas why?

**Comment by ljosberinn** [comment, marked as answer]
reported in #2106, solved by #2114 yet unreleased; we discussed this on discord additionally so I'm closing this here :)

**Reply by Indy-Dev** [reply]
Thanks for your help. Save me lots of time.
